docs/topology/concepts/config.md

```diff
@@ -11,10 +13,13 @@ Config is used to associate config items with your components.
 
 ## Config
 
-| Field       | Description              | Scheme              | Required |
-| ----------- | ------------------------ | ------------------- | -------- |
-| `id`        | Set unique ID for config | `[]string`          |          |
```

Comment:
I removed the `id` field because it wasn't being used at all. Maybe it should also be removed from the Go struct?